### ISSUE TITLE
pour le sujet des clients android:

### ISSUE THREAD

**Issue by matt8fr** [MT-Eurythmia/Eurythmia-website#1]
Grossam je sais que tu veux parler de multicraft mais il existe plein de client android dont voici la[ liste classer](https://forum.minetest.net/viewtopic.php?f=3&t=16707): légale ou pas

**Comment by Grossam** [comment]
En fait, je préférerais me cantonner au client Minetest, car je n'ai que celui là et je ne connais pas les autres.

Mais ceux qui connaissent d'autres clients, surtout Multicraft (le plus utilisé ?) pourraient ajouter de petites précisions par-ci par-là, avec une présentation spéciale (on peut faire des encadrées en Markdown ?).

**Comment by matt8fr** [comment]
il faut voire avec foxcraft qui utilise worldcraft et connais multicraft.

**Comment by paly2** [comment]
> on peut faire des encadrées en Markdown ?

Non, pas en Markdown natif 😕 Mais des plugins de jekyll, comme [celui-ci](https://github.com/gjtorikian/extended-markdown-filter), permettent d'ajouter cette fonctionnalité.

**Comment by Grossam** [comment]
Et ça s'installe comment un greffon Jekyll ?

**Comment by paly2** [comment]
Ça commence par s'installer avec RubyGems, puis le champ `gems` du fichier de configuration de Jekyll (`_config.yml`) se configure comme ceci pour savoir qu'il doit charger le greffon:
```yaml
gems:
  - extended-markdown-filter
  - jekyll-html-pipeline
```
(En l'occurence, j'ai pris l'exemple du greffon dont j'ai passé le lien, qui est aussi dépendant du greffon `jekyll-html-pipeline`.)
